src/components/App.tsx

```diff
@@ -14,7 +15,9 @@ const App: React.FunctionComponent = () => {
       <Routes>
         <Route path="/" element={<LandingPage />} />
         <Route path="/fun-facts" element={<FunFactPage />} />
-        <Route path="/rovers" element={<RoverSelectorPage />} />
+        <Route path="/rovers" element={<RoverSelectorPage />}/>
+        <Route path="/curiosity" element={<CuriosityMap />}/>
```

Comment:
There should be a space before the closing tag, i.e:
```suggestion
        <Route path="/rovers" element={<RoverSelectorPage />} />
        <Route path="/curiosity" element={<CuriosityMap />} />
```